src/main/java/com/wavefront/WavefrontTraceLoader.java

```diff
@@ -29,16 +31,20 @@ void setupSenders() throws IOException {
     }
     // TODO do we need additional checks here??
     WavefrontSender wavefrontSender;
```

Comment:
wafefrontSender definition can go to else body.

Reply:
Done